frontend/packages/kubevirt-plugin/src/k8s/wrapper/common/k8s-resource-wrapper.ts

```diff
@@ -79,4 +80,22 @@ export abstract class K8sResourceWrapper<
     }
     return (this as any) as SELF;
   };
+
+  addOwnerReferences = (...additionalOwnerReferences) => {
+    if (!getOwnerReferences(this.data)) {
```

Comment:
you can use `ensurePath` function instead

Reply:
`ensurePath` doesn't initialize `ownerReferences` as an array. `getOwnerReferences(this.data)` returns `{}` if there are no owners, resulting in errors inside the array functions

Reply:
please look at the implementation, it is possible to pass the desired structure

Reply:
updated